app/post/controller.js

```diff
@@ -16,6 +16,15 @@ module.exports.create = function (req, res) {
   });
 };
 
+module.exports.setHidden = function (req, res) {
+  Post.findById(req.params.id, function (err, post) {
```

Comment:
Look and see if there is a mongo method to find and modify in on call to the database